crates/store/re_log_encoding/src/codec/arrow.rs

```diff
@@ -8,7 +8,18 @@ pub(crate) fn write_arrow_to_bytes<W: std::io::Write>(
     writer: &mut W,
     batch: &ArrowRecordBatch,
 ) -> Result<(), CodecError> {
-    let mut sw = arrow::ipc::writer::StreamWriter::try_new(writer, batch.schema_ref())
+    re_tracing::profile_function!();
+
+    let mut schema = std::sync::Arc::unwrap_or_clone(batch.schema().clone());
+
+    // Remember when the encoding took place,
+    // for ~accurate latency measurement of e.g. gRPC calls.
+    schema.metadata.insert(
+        re_sorbet::timing_metadata::LAST_ENCODED_AT.to_owned(),
+        re_sorbet::timing_metadata::now_timestamp(),
+    );
```

Comment:
I should maybe move these to `re_grpc_server/client` instead so we don't add them when writing to file… but then again, it's nice to have these latency measurements when e.g. piping data view stdin/stdout, or when using the future `tee` feature

Reply:
agreed, might as well keep them always. It's not adding all that much overhead to the file I reckon anyways
Can you please add a comment here to document this rationale? Lest we visit this later and wonder why